src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -40,16 +39,17 @@ const xyChartMargin = { left: 30, right: 30, top: 0, bottom: 30 }
 
 // how many points to generate for the chart, higher is more accurate but slower
 const CHART_GRANULARITY = 100
-// Generate data for tradeSize and foxHolding
 const tradeSizeData = [...Array(CHART_GRANULARITY).keys()].map(
-  i => i * (CHART_TRADE_SIZE_MAX_USD / (CHART_GRANULARITY - 1)),
+  i => FEE_CURVE_NO_FEE_THRESHOLD_USD + i * (CHART_TRADE_SIZE_MAX_USD / (CHART_GRANULARITY - 1)),
 )
 
 const accessors = {
-  xAccessor: ({ x }: { x: number }) => x,
-  yAccessor: ({ y }: { y: number }) => y,
+  xAccessor: (data: { x: number }) => data?.x,
+  yAccessor: (data: { y: number }) => data?.y,
```

Comment:
either type definition is wrong or data is never undefined

```suggestion
  xAccessor: (data: { x: number }) => data.x,
  yAccessor: (data: { y: number }) => data.y,
```

```suggestion
  xAccessor: (data?: { x: number }) => data?.x,
  yAccessor: (data?: { y: number }) => data?.y,
```

Reply:
Looking at `data` below looks like type are definitely wrong - 4e0d85e54c